python/fapolicy_analyzer/ui/main_window.py

```diff
@@ -14,6 +15,19 @@ def __init__(self):
         self.window.show_all()
 
     def on_destroy(self, *args):
+        # Check backend for unapplied changes
+        if( Changeset().is_empty() == False ):
```

Comment:
It's probably safe to do a falsey check here.
```suggestion
        if not Changeset().is_empty():
```

Reply:
Modifed the conditional as auggested.